app/helpers/application_helper.rb

```diff
@@ -46,6 +46,35 @@ def multiple_with_filter?
     params.key?(:search)
   end
 
+  # this helper should be used to print date time in absolute form
+  # it will also define a title with relative time information
+  # it supports two formats :short and :long
+  # example of long is February 12, 2021 17:13
+  # example of short is Aug 31, 12:52
+  def date_time_absolute(time, format = :short)
+    raise ArgumentError, "unsupported format '#{format}', use :long or :short" unless %w(long short).include?(format.to_s)
+
+    content_tag :span, :title => foreman_relative_date_time_value(time) do
+      foreman_absolute_date_time_value(time, format)
+    end
+  end
+
+  # this helper should be used to print date time in relative form, e.g. "10 days ago",
+  # it will also define a title with absolute time information
+  def date_time_relative(time)
+    content_tag :span, :title => foreman_absolute_date_time_value(time, :long) do
+      foreman_relative_date_time_value(time)
+    end
+  end
+
+  def foreman_absolute_date_time_value(time, format = :short)
```

Comment:
Still some methods that use the Foreman prefix, `data_time_absolute_value` would work for me.